src/planet_coords.jl

```diff
@@ -54,7 +54,7 @@ Find the RA, Dec of Venus on 1992 Dec 20
 
 ```jldoctest
 julia> adstring(planet_coords(DateTime(1992,12,20),2))
-" 02 12 22.7  +12 44 45"
+" 21 00 15.3  -19 09 09"
```

Comment:
Is this change due to https://github.com/JuliaAstro/AstroLib.jl/pull/30/commits/f1ce1be493a934d0aa490fcccd827bbf83e0f583?